R/aggregate_game_stats_kicking.R

```diff
@@ -0,0 +1,186 @@
+#' Summarize Kicking Stats
+#'
+#' @description Build columns that aggregate kicking stats at the game level.
+#'
+#' @param pbp A Data frame of NFL play-by-play data typically loaded with
+#' [load_pbp()] or [build_nflfastR_pbp()].
+#' @param weekly If `TRUE`, returns week-by-week stats, otherwise, stats for
+#' the entire season.
+#'
+#' @examples
+#' \donttest{
+#' pbp <- nflreadr::load_pbp(2021)
+#' calculate_player_stats_kicking(pbp, weekly = TRUE)
+#' }
```

Comment:
Adding this try block prevents CRAN failures when they decide to run examples
```suggestion
#' \donttest{
#' try({
#'     pbp <- nflreadr::load_pbp(2021)
#'     calculate_player_stats_kicking(pbp, weekly = TRUE)
#' })
#' }
```